lua/toggleterm/config.lua

```diff
@@ -29,6 +29,7 @@ local function shade(color, factor) return colors.shade_color(color, factor) end
 --- @field direction  '"horizontal"' | '"vertical"' | '"float"'
 --- @field shading_factor number
 --- @field shell string
+--- @field auto_scroll boolean
```

Comment:
This needs to be added to the terminal class type annotation itself just below other than that this LGTM